<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>813</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="Qwidgt" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QStackedWidget" name="gamewidget">
            <property name="currentIndex">
             <number>3</number>
            </property>
            <widget class="QWidget" name="page">
             <widget class="QLabel" name="label_2">
              <property name="geometry">
               <rect>
                <x>130</x>
                <y>140</y>
                <width>272</width>
                <height>33</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;楷体&quot;;
font: 20pt &quot;楷体&quot;rgb(0, 0, 0);</string>
              </property>
              <property name="text">
               <string>请输入游玩人数：</string>
              </property>
             </widget>
             <widget class="QPushButton" name="startbtn">
              <property name="geometry">
               <rect>
                <x>270</x>
                <y>240</y>
                <width>93</width>
                <height>28</height>
               </rect>
              </property>
              <property name="text">
               <string>开始游戏</string>
              </property>
             </widget>
             <widget class="QComboBox" name="numbox">
              <property name="geometry">
               <rect>
                <x>400</x>
                <y>150</y>
                <width>46</width>
                <height>21</height>
               </rect>
              </property>
              <item>
               <property name="text">
                <string>3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>6</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>7</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>9</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10</string>
               </property>
              </item>
             </widget>
            </widget>
            <widget class="QWidget" name="jxpage">
             <widget class="QPushButton" name="jxBtn">
              <property name="geometry">
               <rect>
                <x>190</x>
                <y>220</y>
                <width>93</width>
                <height>28</height>
               </rect>
              </property>
              <property name="text">
               <string>竞选村长</string>
              </property>
             </widget>
             <widget class="QPushButton" name="bjxbtn">
              <property name="geometry">
               <rect>
                <x>320</x>
                <y>220</y>
                <width>93</width>
                <height>28</height>
               </rect>
              </property>
              <property name="text">
               <string>不竞选村长</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="eyewidget">
             <widget class="QPushButton" name="openeye">
              <property name="geometry">
               <rect>
                <x>200</x>
                <y>260</y>
                <width>93</width>
                <height>28</height>
               </rect>
              </property>
              <property name="text">
               <string>睁眼</string>
              </property>
             </widget>
             <widget class="QPushButton" name="eyeclose">
              <property name="geometry">
               <rect>
                <x>340</x>
                <y>260</y>
                <width>93</width>
                <height>28</height>
               </rect>
              </property>
              <property name="text">
               <string>继续闭着</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="afterjx">
             <widget class="QTableWidget" name="information">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>671</width>
                <height>201</height>
               </rect>
              </property>
             </widget>
             <widget class="QTextEdit" name="duihua">
              <property name="geometry">
               <rect>
                <x>410</x>
                <y>200</y>
                <width>301</width>
                <height>401</height>
               </rect>
              </property>
             </widget>
             <widget class="QLineEdit" name="lineEdit">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>550</y>
                <width>391</width>
                <height>21</height>
               </rect>
              </property>
             </widget>
             <widget class="QPushButton" name="speakbtn">
              <property name="geometry">
               <rect>
                <x>50</x>
                <y>580</y>
                <width>93</width>
                <height>28</height>
               </rect>
              </property>
              <property name="text">
               <string>确认发言</string>
              </property>
             </widget>
             <widget class="QPushButton" name="notspeakbtn">
              <property name="geometry">
               <rect>
                <x>260</x>
                <y>580</y>
                <width>93</width>
                <height>28</height>
               </rect>
              </property>
              <property name="text">
               <string>进入投票</string>
              </property>
             </widget>
             <widget class="QPushButton" name="afspeak">
              <property name="geometry">
               <rect>
                <x>160</x>
                <y>580</y>
                <width>93</width>
                <height>28</height>
               </rect>
              </property>
              <property name="text">
               <string>确认发言</string>
              </property>
             </widget>
             <widget class="QPushButton" name="nextbtn">
              <property name="geometry">
               <rect>
                <x>410</x>
                <y>600</y>
                <width>93</width>
                <height>28</height>
               </rect>
              </property>
              <property name="text">
               <string>Next</string>
              </property>
             </widget>
             <widget class="QPushButton" name="nightbtn">
              <property name="geometry">
               <rect>
                <x>500</x>
                <y>600</y>
                <width>93</width>
                <height>28</height>
               </rect>
              </property>
              <property name="text">
               <string>night</string>
              </property>
             </widget>
             <widget class="QPushButton" name="yiyanbtn">
              <property name="geometry">
               <rect>
                <x>160</x>
                <y>580</y>
                <width>93</width>
                <height>28</height>
               </rect>
              </property>
              <property name="text">
               <string>发表遗言</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="jxcunz">
             <widget class="QPushButton" name="backbtn">
              <property name="geometry">
               <rect>
                <x>490</x>
                <y>220</y>
                <width>93</width>
                <height>28</height>
               </rect>
              </property>
              <property name="text">
               <string>返回</string>
              </property>
             </widget>
             <widget class="QGroupBox" name="groupBox">
              <property name="geometry">
               <rect>
                <x>180</x>
                <y>170</y>
                <width>142</width>
                <height>41</height>
               </rect>
              </property>
              <property name="title">
               <string/>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>请玩家进行投票</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QComboBox" name="jxplayer">
              <property name="geometry">
               <rect>
                <x>210</x>
                <y>240</y>
                <width>87</width>
                <height>22</height>
               </rect>
              </property>
             </widget>
             <widget class="QPushButton" name="votebtn">
              <property name="geometry">
               <rect>
                <x>150</x>
                <y>300</y>
                <width>93</width>
                <height>28</height>
               </rect>
              </property>
              <property name="text">
               <string>投票</string>
              </property>
             </widget>
             <widget class="QPushButton" name="cancelvote">
              <property name="geometry">
               <rect>
                <x>270</x>
                <y>300</y>
                <width>93</width>
                <height>28</height>
               </rect>
              </property>
              <property name="text">
               <string>弃票</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="orderwidget">
             <widget class="QRadioButton" name="leftradioButton">
              <property name="geometry">
               <rect>
                <x>190</x>
                <y>150</y>
                <width>115</width>
                <height>19</height>
               </rect>
              </property>
              <property name="text">
               <string>left</string>
              </property>
             </widget>
             <widget class="QRadioButton" name="rightradioButton">
              <property name="geometry">
               <rect>
                <x>290</x>
                <y>150</y>
                <width>115</width>
                <height>19</height>
               </rect>
              </property>
              <property name="text">
               <string>right</string>
              </property>
             </widget>
             <widget class="QPushButton" name="orderbtn">
              <property name="geometry">
               <rect>
                <x>230</x>
                <y>200</y>
                <width>93</width>
                <height>28</height>
               </rect>
              </property>
              <property name="text">
               <string>确定</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="voteout">
             <widget class="QLabel" name="label_3">
              <property name="geometry">
               <rect>
                <x>180</x>
                <y>220</y>
                <width>111</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>请玩家进行投票</string>
              </property>
             </widget>
             <widget class="QComboBox" name="voteoutbox">
              <property name="geometry">
               <rect>
                <x>190</x>
                <y>250</y>
                <width>87</width>
                <height>22</height>
               </rect>
              </property>
             </widget>
             <widget class="QPushButton" name="voteoutbtn">
              <property name="geometry">
               <rect>
                <x>120</x>
                <y>300</y>
                <width>93</width>
                <height>28</height>
               </rect>
              </property>
              <property name="text">
               <string>投票</string>
              </property>
             </widget>
             <widget class="QPushButton" name="novoteoutbtn">
              <property name="geometry">
               <rect>
                <x>250</x>
                <y>300</y>
                <width>93</width>
                <height>28</height>
               </rect>
              </property>
              <property name="text">
               <string>弃权</string>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>827</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="startmenu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionclose"/>
   </widget>
   <addaction name="startmenu"/>
  </widget>
  <action name="actionclose">
   <property name="text">
    <string>结束游戏</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
